crates/iota-genesis-builder/src/lib.rs

```diff
@@ -149,11 +149,25 @@ impl Builder {
         }
     }
 
+    /// Return an iterator of migration objects if this genesis is with
+    /// migration objects.
+    pub fn tx_migration_objects(&self) -> impl Iterator<Item = Object> + '_ {
+        self.migration_tx_data
+            .as_ref()
+            .map(|tx_data| tx_data.get_objects())
+            .into_iter()
+            .flatten()
+    }
+
+    /// Set the genesis delegation to be a `OneToAll` kind and indicates the
+    /// delegator address.
```

Comment:
```suggestion
    /// Set the genesis delegation to be a `OneToAll` kind and indicate the
    /// delegator address.
```

Reply:
you chose the term "indicate", because we're not really setting a delegator address?